addons/binding/org.openhab.binding.gpio/src/main/java/org/openhab/binding/gpio/GPIOBindingConstants.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.gpio;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link GPIOBinding} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author nils-developersinblue - Initial contribution
+ */
+public class GPIOBindingConstants {
+
+    public static final String BINDING_ID = "gpio";
+
+    public final static ThingTypeUID THING_TYPE_PIGPIO_LOCAL_BRIDGE = new ThingTypeUID(BINDING_ID,
```

Comment:
Non-preferred order of modifiers (It was like this in a template, which has been changed recently) correct order is `public static final` See also: http://stackoverflow.com/questions/16731240/what-is-a-reasonable-order-of-java-modifiers-abstract-final-public-static-e#16732059